GUIDELINES.md

```diff
@@ -0,0 +1,104 @@
+# GUIDELINES REACT JS
+
+> Here you can find the Axa Front End Community recommandations for developping React Application
+
+- [GUIDELINES REACT JS](#guidelines-react-js)
+  - [Philosophy](#philosophy)
+  - [Organize your application](#organize-your-application)
+    - [General structure](#general-structure)
+    - [Naming convention](#naming-convention)
+  - [Stack :](#stack-)
+    - [ES Lint Configuration](#es-lint-configuration)
+    - [Recommanded Tools](#recommanded-tools)
+
+## Philosophy
+
+When you developping React application you **MUST** follow somes principles :
```

Comment:
"When you develop React application(s) you **MUST** follow some principles :"

Reply:
yes merci